.github/workflows/e2e.yml

```diff
@@ -91,7 +91,6 @@ jobs:
           fi
           time dfx deploy
           time dfx canister call smoke_backend greet '("fire")'
-          time curl --fail http://localhost:8000/sample-asset.txt?canisterId=$(dfx canister id smoke_frontend)
```

Comment:
Can we create this project with a frontend?  Ensuring that a basic project can serve assets is an important part of the "smoke" test.